### DISCUSSION TITLE
[HELP] CUDA support

### DISCUSSION THREAD

**Discussion by JPaparone-inria** [cnr-isti-vclab/meshlab#914] [Q&A]
Hi ! First of all, thanks a lot for this amazing software !
I've seen in your tutorials it is possible to enable GPU rendering on NVidia cards to speed up the display of large meshes. However, the steps proposed to do so rely on tools provided by the Windows GUI. Is it possible (and if so, how ?) to access some panel in which settings the CUDA parameters (in a Blender sort of way, for instance).
Thanks a lot !

**Comment by alemuntoni** [comment]
Sorry for the late reply!
Right now it is not possible to do this. This kind of feature needs to be developed from scratch, since there is no code infrastructure in MeshLab to make it an "easy feature request". Therefore I don't think that this is going to be implemented in the near future. 
However, if there are some volunteers that want to help on the development and send PRs on this (or any other) topic, they are very welcome!
